src/docs/cursors.dox

```diff
@@ -21,26 +21,6 @@ See the following for more details:
 - @ref cursor_log
 - @ref cursor_join
 
-@section cursor_projections Projections
```

Comment:
This was removed due to git problems. It is part of wt-12472, not this PR. Put back.